server/src-lib/Hasura/Class.hs

```diff
@@ -0,0 +1,84 @@
+module Hasura.Class
```

Comment:
Maybe this could use a more specific name. Something like `Hasura.Metadata.Class`?